packages/ui/src/components/flag/flag.shared.tsx

```diff
@@ -0,0 +1 @@
+export type FlagAlignment = 'top' | 'middle' | 'bottom';
```

Comment:
I tried adding these in a `types.ts` file but struggled with the CI failing with `flag.types.ts` is not listed within the file list of the project. 

I tried to configure `tsconfig` but couldn't get it working. 

@fbwoolf I see this is what you did to share types in icons. Did you hit similar issues? 

I thought that just adding `types.ts` to `tsconfig.json` should work

Reply:
Basically, with ts and builds, any files with extension `.native` or `.web` are included in those separate builds AND `.shared` are included in both.

Reply:
I think what you have is fine tbh, but could just be `flag.shared.ts`.

Reply:
Thanks @fbwoolf . I did try adding it to the includes yesterday but I couldn't get it working. 

I'll investigate more as I add more components to see if I can move it but at least now its doing what we want - sharing types